IBM/ubiquity-k8s
# Fix/ub 624 flex should not write log into flex driver director

Handle flex logrotate internal
    configurable for FLEX-LOG-DIR in ubiquity-configmap.yml and ubiquity_installer.conf, default is /var/log, customer need to make sure Flex-Log-dir exist in the minors
    configurable for FLEX-LOG-ROTATE-MAXSIZE in ubiquity-configmap.yml, default is 50MB


How to test:
    1. check the default log path: /var/log(exist in all flex pods, and nodes), logs will rotate if > 50 MB, will keep 5 compressed log file
    2. can change log path to example /var/log/ubiquity in ubiquity_install.conf and configmap, need to make sure this path existed(in all nodes, also for controller-manager-pod) before install
    3. can change log rotate size from 50MB to others in ubiquity-configmap
    4. flex binary will create the log dir (ex: /var/log/ubiquity) if it not exist
    5. flex logs not sit on k8s plugin directory


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ibm/ubiquity-k8s/166)
